### PR TITLE
[FR] Translation of chapter 2 & event + Review of chapters 0 & 5

### PR DESCRIPTION
**Pull request by lbourdois**
Hi @lewtun 
Here is the translation of the chapter 2 into French + a review of chapters 0 and 5 which contained typo + translation of event.

### CODE REVIEW

**Review by lewtun — APPROVED** [review]
Thank for contributing a whole chapter and making the fixes in the others 🤯 . This looks really great - once the CI passes, I'll merge it!

**Comment by lbourdois** [comment]
I forgot to update the toctree file to add the event, that must be why it didn't pass the checks

**Comment by lewtun** [comment]
If you want to fix the code formatting locally, you can run:

```
python -m pip install -r requirements.txt
```

followed by

```
make style
```

**Comment by lbourdois** [comment]
I hope everything is good now

File: chapters/fr/_toctree.yml [hunk 1]
```diff
@@ -3,16 +3,36 @@
   - local: chapter0/1
     title: Introduction
 
+ title: 2. Utilisation de 🤗 Transformers
```
Comment (lewtun) [inline chapters/fr/_toctree.yml:6]: I think we need a `-` here :)

```suggestion
- title: 2. Utilisation de 🤗 Transformers
```

Comment (lbourdois) [reply chapters/fr/_toctree.yml]: Done

Comment (lewtun) [reply chapters/fr/_toctree.yml]: Thank you!

**Comment by lbourdois** [comment]
image
I quickly consulted the final rendering to see what it looks like:
- in red on the image you can see that the word "token" has not been italicized. Markdown tags don't seem to work on titles/subtitles. Can I do something about it by using HTML tags for example or can't I do anything?
- in orange, the apostrophe is not displayed properly
Is there any way I can fix this in a future PR or is this something only the Hugging Face team has a hand in?

**Comment by lewtun** [comment]
Gently pinging @mishig25 for advice on the above frontend questions 🙏

**Comment by lbourdois** [comment]
In the same logic, I don't know if what is circled in orange in the picture could be translated in each language:

image

**Pull request merged**